src/intl/en/page-wallets.json

```diff
@@ -17,8 +17,8 @@
   "page-wallets-desc-3": "Your wallet is only a tool for managing your Ethereum account. That means you can swap wallet providers at any time. Many wallets also let you manage several Ethereum accounts from one application.",
   "page-wallets-desc-4": "That's because wallets don't have custody of your funds, you do. They're just a tool for managing what's really yours.",
   "page-wallets-description": "Ethereum wallets are applications that let you interact with your Ethereum account. Think of it like an internet banking app – without the bank. Your wallet lets you read your balance, send transactions and connect to applications.",
-  "page-wallets-desktop": "Desktop applications if you prefer to manage your funds via MacOS, windows or linux",
-  "page-wallets-ethereum-wallet": "A wallet is a product that lets you manage your Ethereum account, like view your account balance, send transactions, and more.",
+  "page-wallets-desktop": "Desktop applications if you prefer to manage your funds via MacOS, Windows or Linux",
```

Comment:
while we're in here, I would suggest make it `macOS`

Reply:
Yep that's good with me. Had to check on Apple's website because I don't have a mac haha.